Get an error message  when trying to do : "Check out the project and do dependency resolution"

<!-- 
When reporting bugs and requesting assistance, please include:
* the version of CoreDNS you are using
* your Corefile
* logs, if applicable

Thanks for using CoreDNS!
-->
Build CoreDNS from scratch with the following steps
1. Install golang
$ go version
go version go1.11.2 linux/amd64

2.  Check out the project and do dependency resolution
$ go get github.com/coredns/coredns
   github.com/coredns/coredns/core/dnsserver
go/src/github.com/coredns/coredns/core/dnsserver/register.go:252:5: cannot use new(Server) (type *Server) as type caddy.GracefulServer in assignment:
        *Server does not implement caddy.GracefulServer (missing WrapListener method)

3. git master
$ git log
commit 18f25dbef3a33092925e48e7d7f9c8d8f744cd05

--- a/README.md
+++ b/README.md
@@ -44,15 +44,17 @@ out-of-tree plugins.
 
 ## Compilation from Source
 
-Check out the project and do dependency resolution with:
+Check out the project:
 
 ~~~
-% go get github.com/coredns/coredns
+% git clone https://github.com/coredns/coredns
 ~~~
 
-Some of the dependencies require Go version 1.9 or later.
+We vendor most (not all!) packages. This is mostly because vendoring isn't a perfect solution (in
+Go). We don't vendor `mholt/caddy` and `miekg/dns` for instance. Using `make` will pull down these
+dependencies and check out the correct version as well.
 
-We vendor most (not all!) packages. Building from scratch is easiest, by just using `make`:
+Next just run `make`:
 
 ~~~
 % make

